apps/right-to-rent-check/views/declaration.html

```diff
@@ -0,0 +1,11 @@
+{{<partials-page}}
+  {{$page-content}}
+    <p>By submitting this request:</p>
+    <ul class="no-margin-bottom">
+    <li id={{declarerID}}>{{declarer}}</li>
```

Comment:
indentation

Reply:
Will `id={{declarerID}}` render with quotes around the value, like ... `id="{{declarerID}}"`

Reply:
it does as expected e.g. `id="landlord"`